mdn_theme/mdn/layout.html

```diff
@@ -1,56 +1,55 @@
-<!DOCTYPE html><html class="redesign no-js" dir="ltr" lang="en-US"><head prefix="og: http://ogp.me/ns#">
+<html data-ffo-opensanslight="false" data-ffo-opensans="false" lang="None" dir="ltr" class="redesign no-js"><head>
   <meta charset="utf-8">
   <meta content="IE=Edge" http-equiv="X-UA-Compatible">
   <script>(function(d) { d.className = d.className.replace(/\bno-js/, ''); })(document.documentElement);</script>
-    <title>{{ docstitle|e }} &amp;mdash; MDN</title>
+  
+  <title>{{ docstitle }} - MDN</title>
 
   <meta content="width=device-width, initial-scale=1" name="viewport">
   <meta content="index, follow" name="robots">
-  <link href="https://developer.mozilla.org/" rel="home"></link>
-  <link href="https://developer.mozilla.org/#copyright" rel="copyright"></link>
 
-      <link href="https://developer.mozilla.org/static/styles/font-awesome.css" media="all" rel="stylesheet"></link>
-<link href="https://developer.mozilla.org/static/styles/main.css" media="all" rel="stylesheet"></link>
+  
+    <link type="text/css" href="https://developer.mozilla.org/static/build/styles/mdn.65be1fdadb40.css" rel="stylesheet"></link>
 
-  <!-- common social tags -->
-    <meta content="website" property="og:type">
-  <meta content="http://testserver/static/img/opengraph-logo.png" property="og:image">
-  <meta content="Mozilla Developer Network" property="og:site_name">
-  <meta content="summary" name="twitter:card">
-  <meta content="http://testserver/static/img/opengraph-logo.png" name="twitter:image">
-  <meta content="@NewOnMDN" name="twitter:site">
-  <meta content="@NewOnMDN" name="twitter:creator">
-
-  <link href="https://developer-local.allizom.org/en-US/search/xml" rel="search" title="Mozilla Developer Network" type="application/opensearchdescription+xml"></link>
+    
+  
 
   <!-- third-generation iPad with high-resolution Retina display: -->
-  <link href="https://developer.mozilla.org/static/img/favicon144.png" rel="apple-touch-icon-precomposed" sizes="144x144"></link>
+  <link href="https://developer.mozilla.org/static/img/favicon144.a6e4162070f4.png" rel="apple-touch-icon-precomposed" sizes="144x144"></link>
```

Comment:
The thing stripping the doctype might also be adding the closing tags for `<link>`. They're not required by HTML5. Not breaking anything either, so not launch blocking.